src/responder/common/cache_req/cache_req_data.c

```diff
@@ -84,7 +84,15 @@ cache_req_data_create(TALLOC_CTX *mem_ctx,
 
```

Comment:
While we are here, let's also make `struct cache_req_data *input` a pointer to the const data.